outputs.tf

```diff
@@ -11,7 +11,10 @@ output "dns_name" {
 }
 
 output "mount_target_ids" {
-  value = aws_efs_mount_target.default.*.id
+  value = {
+    for fsmt in aws_efs_mount_target.default :
+    fsmt.availability_zone_name => fsmt.mount_target_dns_name
```

Comment:
@JonRoma Is this correct? I'm not sure what this change is about. Can you please explain to me what you are trying to accomplish with lines 14-17?

It looks like before the code was returning a list of EFS mount targets IDs, and now it is returning a map of AZ pointing to EFS mount targets DNS names. Is that right? If so I think this should have a different output label (name), and be documented in the README.md.